web/vital_records/models.py

```diff
@@ -30,6 +31,68 @@ class VitalRecordsRequest(models.Model):
         ("domestic_partner", "Domestic partner"),
     ]
 
+    COUNTY_CHOICES = [
+        ("", "Select county"),
+        ("01", "Alameda"),
+        ("02", "Alpine"),
+        ("03", "Amador"),
+        ("04", "Butte"),
+        ("05", "Calaveras"),
+        ("06", "Colusa"),
+        ("07", "Contra Costa"),
+        ("08", "Del Norte"),
+        ("09", "El Dorado"),
+        ("10", "Fresno"),
+        ("11", "Glenn"),
+        ("12", "Humboldt"),
+        ("13", "Imperial"),
+        ("14", "Inyo"),
+        ("15", "Kern"),
+        ("16", "Kings"),
+        ("17", "Lake"),
+        ("18", "Lassen"),
+        ("19", "Los Angeles"),
+        ("20", "Madera"),
+        ("21", "Marin"),
+        ("22", "Mariposa"),
+        ("23", "Mendocino"),
+        ("24", "Merced"),
+        ("25", "Modoc"),
+        ("26", "Mono"),
+        ("27", "Monterey"),
+        ("28", "Napa"),
+        ("29", "Nevada"),
+        ("30", "Orange"),
+        ("31", "Placer"),
+        ("32", "Plumas"),
+        ("33", "Riverside"),
+        ("34", "Sacramento"),
+        ("35", "San Benito"),
+        ("36", "San Bernardino"),
+        ("37", "San Diego"),
+        ("38", "San Francisco"),
+        ("39", "San Joaquin"),
+        ("40", "San Luis Obispo"),
+        ("41", "San Mateo"),
+        ("42", "Santa Barbara"),
+        ("43", "Santa Clara"),
+        ("44", "Santa Cruz"),
+        ("45", "Shasta"),
+        ("46", "Sierra"),
+        ("47", "Siskiyou"),
+        ("48", "Solano"),
+        ("49", "Sonoma"),
+        ("50", "Stanislaus"),
+        ("51", "Sutter"),
+        ("52", "Tehama"),
+        ("53", "Trinity"),
+        ("54", "Tulare"),
+        ("55", "Tuolumne"),
+        ("56", "Ventura"),
+        ("57", "Yolo"),
+        ("58", "Yuba"),
+    ]
+
```

Comment:
This data format comes from the JSON that @thekaveman posted in the dev channel, and corresponds to what the final PDF needs. But I removed 3 invalid fields: Unknown, Out of state and Foreign Country, because we do not want people to select those. Thoughts? @thekaveman @mrose914